EXAMPLE/knx_iot_wakeful_main.c

```diff
@@ -308,6 +308,39 @@ int main(void)
 				 sn[5]);
 		app_set_serial_number(serial_number_str);
 	}
+	
+	char pwd[33];
+	error = knx_get_stored_password(pwd);
+	if (error)
+	{
+		PRINT_APP("Error: Stored password not found! Using default value...\n");
+		PRINT_APP(
+			"Please create the data file using knx-gen-data and flash it with chilictl in order to fix this issue.\n");
+	}
+	else
+	{
+		oc_spake_set_password(pwd);
+	}
+	
+	uint8_t salt[32], rand[32];
+	uint32_t it;
+	mbedtls_mpi w0;
+	mbedtls_ecp_point L;
+	mbedtls_mpi_init(&w0);
+	mbedtls_ecp_point_init(&L);
+	error = knx_get_stored_spake(salt, rand, &it, &w0, &L);
+	if (error)
+	{
+		PRINT_APP("Error: Stored spake record not found! Using runtime generated values\n");
+		PRINT_APP(
+			"Please create the data file using knx-gen-data and flash it with chilictl in order to fix this issue.\n");
+	}
+	else
+	{
+		oc_spake_set_parameters(rand, salt, it, w0, L);
+		mbedtls_mpi_free(&w0);
+		mbedtls_ecp_point_free(&L);
+	}
 
 	/* set the application callbacks */
 	oc_set_hostname_cb(hostname_cb, NULL);
```

Comment:
note: the build is not working

Reply:
We know! Gotta wait for a public SDK release, which is dependant on Skyler's current task